NewForce-at-Mountwest/bangazon-site-take-2-fluffy
# order history accessible from personal profile and displays history c…

# Description

Users can now access their order history from their personal profile.

## Related Ticket(s)
17

## Steps to Test
Add and commit on your branch. Pull down the branch sb-product-details. Run your server.
Navigate to the user personal profile page by clicking on the user's name in the nav bar.
Click the "Order History" link on the side nav bar. This should take you to an index page of the user's orders. (Make sure you have multiple orders in your history!) This should display the date created/completed and the payment account number.
You should be able to click the "details" button to view details of the order.


# Checklist:
